packages/bierzo-wallet/src/logic/transactions/types/BwSendTransaction/ui/SendTxRow/Details.tsx

```diff
@@ -1,42 +1,96 @@
 import { Address } from "@iov/bcp";
-import { Block, Typography } from "medulas-react-components";
+import { Block, makeStyles, Typography } from "medulas-react-components";
 import * as React from "react";
+import { amountToString } from "ui-logic";
 
 import BlockchainAddress from "../../../../../../components/BlockchainAddress";
 import { ProcessedSendTransaction } from "../../../../../../store/notifications";
+import { formatTime } from "../../../../../../utils/date";
+
+const useStyles = makeStyles({
+  rowToggle: {
+    cursor: "pointer",
+  },
+  amountFeeTo: {
+    "&::before": {
+      content: `"-"`,
+    },
+  },
+});
```

Comment:
Why so complex solution instead of simply adding minus sign? Also it will be hard to read page source code with this class (it will take time to understand where minus sign is coming from). But if this is really required so why don't you want to use same solution in packages/bierzo-wallet/src/logic/transactions/types/BwRegisterUsernameTx/ui/TransactionRow/Details.tsx?